src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DiskStationProxyBase.cs

```diff
@@ -172,40 +112,62 @@ private string GenerateSessionCacheKey(DownloadStationSettings settings)
 
         private string AuthenticateClient(DownloadStationSettings settings)
         {
-            var arguments = new Dictionary<string, object>
-            {
-                 { "api", "SYNO.API.Auth" },
-                 { "version", "1" },
-                 { "method", "login" },
-                 { "account", settings.Username },
-                 { "passwd", settings.Password },
-                 { "format", "sid" },
-                 { "session", "DownloadStation" },
-             };
-
             var authInfo = GetApiInfo(DiskStationApi.Auth, settings);
 
-            var requestBuilder = BuildRequest(settings, authInfo, HttpMethod.GET);
+            var requestBuilder = BuildRequest(settings, authInfo, "login", 2);
```

Comment:
Sorry, you are right, I need version 2 in order to be able to use sid. According to documentation v2 is available since DSM 4.1. I wonder how many people have not been able to update theirs DiskStation...